backend/controllers/attendance/attendance.ts

```diff
@@ -62,28 +51,22 @@ export const attendanceFactory = (whereaboutsApi) => {
   }
 
   const getAbsenceReasons = async (context) => {
-    const [absenceReasons, absenceReasonsV2] = await Promise.all([
-      await whereaboutsApi.getAbsenceReasons(context),
-      await whereaboutsApi.getAbsenceReasonsV2(context),
-    ])
-    const { paidReasons, unpaidReasons, triggersIEPWarning } = absenceReasons
-    const { triggersAbsentSubReason, paidSubReasons, unpaidSubReasons } = absenceReasonsV2
-    const mapToAbsentReason = absentReasonMapper(absenceReasons)
-
-    const approvedCourse = {
-      value: 'ApprovedCourse',
-      name: 'Approved course',
-    }
-
-    const paidReasonsExcluding = (name) =>
-      paidReasons
-        .filter((reason) => reason !== name)
-        .map(mapToAbsentReason)
-        .sort(sortByName)
+    const absenceReasons = await whereaboutsApi.getAbsenceReasonsV2(context)
+    const {
+      paidReasons,
+      unpaidReasons,
+      triggersIEPWarning,
+      triggersAbsentSubReason,
+      paidSubReasons,
+      unpaidSubReasons,
+    } = absenceReasons
 
     return {
-      paidReasons: [approvedCourse, ...paidReasonsExcluding(approvedCourse.value)],
-      unpaidReasons: unpaidReasons.map(mapToAbsentReason).sort(sortByName),
+      paidReasons: paidReasons.map((r) => ({ value: r.code, name: r.name })),
+      unpaidReasons: unpaidReasons.map((r) => ({
+        value: r.code,
+        name: triggersIEPWarning.includes(r.code) ? `${r.name} - incentive level warning` : r.name,
```

Comment:
temporarily have to add in some `incentive level warning` message until we change front end to have IEP radios instead